routemaster/models/subscriber.rb

```diff
@@ -79,6 +79,28 @@ def uuid
       _attributes['uuid']
     end
 
+    def health_points
+      _attributes.fetch('health_points', '100').to_i
+    end
+
+    def change_health_by(offset)
+      new_value = _redis_lua_run(
+        'subscriber_change_health_by',
+        keys: [_key],
+        argv: [offset]
+      )
+      @_attributes = nil
+      new_value
+    end
+
+    def last_attempted_at
+      _attributes['last_attempted_at']&.to_i
```

Comment:
Yeah, just saying it's a little obscure when reading the source here. Tests aren't (good) docs :)